include/seqan/sequence/string_set_dependent_tight.h

```diff
@@ -91,7 +91,7 @@ struct Dependent;
 
 // StringSet with individual sequences in a tight string of string pointers and corr. IDs
 template <typename TString>
-class StringSet<TString, Dependent<Tight> >
+class /*[[deprecated]]*/ StringSet<TString, Dependent<Tight> >
```

Comment:
Well I can remove the deprecation of course.. but as I said changing the default will require some thought and work. Maybe we can talk this through on monday.